src/controllers/controller.bar.js

```diff
@@ -219,6 +219,7 @@ export default class BarController extends DatasetController {
 
 	initialize() {
 		const me = this;
+		me.enableOptionSharing = true;
```

Comment:
any reason we don't make this private?

Reply:
I'd think extensions would want to be setting this

Reply:
Does this need to be in the typedefs at all?

Reply:
Added